.github/workflows/tests.yml

```diff
@@ -164,15 +164,15 @@ jobs:
           echo ""
 
       - name: Setup release Postgres apt repo
-        if: matrix.postgres != '17'
+        if: matrix.postgres != '18'
         run: |
           sudo apt-get install -y wget gnupg
           sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
           wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
           sudo apt-get update -y -qq --fix-missing
 
       - name: Setup beta Postgres apt repo
-        if: matrix.postgres == '17'
+        if: matrix.postgres == '18'
```

Comment:
queue this up for the next version.  The postgres folks seem to put their beta packages in the same place, so we can just carry this along with us so we don't have to re-learn it again next year.